### PR TITLE
vulnerability fixes

### PR DESCRIPTION
**Pull request by akram-r**
updated rollup version for vulnerability fixes

### CODE REVIEW

**Review by somyarocketium — APPROVED** [review]

**Comment by somyarocketium** [comment]
@akram-r can you upgrade the package.json version too ? Will help us in deploying

File: package.json
Comment (venusgazer) [inline package.json:1]: Do we not bump up the versions for react-alignment-guides during vulnerability fixes?

Comment (akram-r) [reply package.json:1]: done updated to 1.0.5

**Comment by akram-r** [comment]
> @akram-r can you upgrade the package.json version too ? Will help us in deploying

done @somyarocketium  bumped to 1.0.5

**Review by venusgazer — APPROVED** [review]

**Review by somyarocketium — APPROVED** [review]

**Pull request merged**